### DISCUSSION TITLE
Additional button click at the end of a shape

### DISCUSSION THREAD

**Discussion by izlence** [jersou/mouse-actions#11] [Ideas]
Lately I am playing with mouse-actions occasionally. 
I like the idea you have GUI and background versions. I can define the shapes using GUI and than I keep running the minimal background version which uses so little memory.
  
Comparing it with easystroke, mouse-actions uses so less memory. 

Being a big fan of easystroke and having almost 200 gestures in the list, I would like to see the extra button click at the end of the shapes. By using this, I am able to define the same shape multiple times, by adding extra button click.

To make it clear what I mean, l am adding a screenshot...

![image](https://github.com/jersou/mouse-actions/assets/10724137/9e9271ba-d75c-4ea7-86d8-8563d2977f35)

**Comment by jersou** [comment]
Thank you for your message ! And thanks to [Rust](https://www.rust-lang.org/) for the low CPU/Ram usage !

If I understand the request correctly, it's to be able to have different buttons for the gestures.
If so, it's already on the TODO list at the end of the Readme (this could be better written): 
> change config : if shape → (no need button in binding) or (rm shape_btn : several button for shape event is then possible)
